docs/en/query_language/select.md

```diff
@@ -964,11 +964,11 @@ External sorting works much less effectively than sorting in RAM.
 
 ### SELECT Clause {#select-select}
 
-[Expressions](syntax.md#syntax-expressions) that specified in the `SELECT` clause are analyzed after the calculations for all the clauses listed above are completed. More specifically, expressions are analyzed that are above the aggregate functions, if there are any aggregate functions. The aggregate functions and everything below them are calculated during aggregation (`GROUP BY`). These expressions work as if they are applied to separate rows in the result.
+[Expressions](syntax.md#syntax-expressions) specified in the `SELECT` clause are analyzed after the calculations for all the clauses listed above are completed. More specifically, expressions above the aggregate functions are analyzed, provided there are aggregate functions. The aggregate functions and everything below them are calculated during aggregation (`GROUP BY`). These expressions work as if they were applied to separate rows in the result.
```

Comment:
we'd better rewrite it, because it's not really obvious what "above" and "below" means here